Engines/Wine/Settings/always offscreen/script.js

```diff
@@ -4,45 +4,50 @@ include("engines.wine.plugins.regedit");
 /**
  * setting to set always offscreen
 */
-var settingImplementation = {
-    _options: [tr("Default"), tr("Disabled"), tr("Enabled")],
-    // values which are written into the registry, do not translate!
-    _registryValues: ["", "disabled", "enabled"],
-    getText: function () {
+class AlwaysOffscreenSetting {
+    constructor() {
+        this.options = [tr("Default"), tr("Disabled"), tr("Enabled")];
+        // values which are written into the registry, do not translate!
+        this.registryValues = ["", "disabled", "enabled"];
+    }
+
+    getText() {
         return tr("Always offscreen");
-    },
-    getOptions: function () {
-        return this._options;
-    },
-    getCurrentOption: function (container) {
-        var currentValue = new Wine()
+    }
+
+    getOptions() {
```

Comment:
I'm not sure whether this method is actually necessary. If we're lucky Graal.js automatically maps the getter method to the `options` field accessor.

Reply:
Can you try?